source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,30 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-07-14
+
+### Highlights
+
+#### Granular Data Retention Policies
```

Comment:
Do we want to mention this known issue Known issue: https://mattermost.atlassian.net/browse/MM-36574?

Reply:
Yeah I think that makes sense.  Added inline suggestion below.